examples/tof-viewer/src/ADIToFRecorder.cpp

```diff
@@ -263,8 +285,6 @@ void ADIToFRecorder::recordThread() {
             std::ofstream recordFileRaw;
             recordFileRaw.open(fileNameRawFrame,
                                std::ios::binary | std::ofstream::trunc);
-            recordFileRaw.write(reinterpret_cast<const char *>(rawData),
-                                static_cast<int>(captures * size));
             recordFileRaw.close();
```

Comment:
Nothing happens between opening and closing the file so it's better to remove the open() and close() calls.